src/Parser/MarksmanshipHunter/Modules/Features/TimeFocusCapped.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Icon from 'common/Icon';
+
+import FocusTracker from '../FocusChart/FocusTracker';
+
+class TimeFocusCapped extends Module {
+  static dependencies = {
+    focusTracker: FocusTracker,
+  };
+
+  getTotalWaste(){
+    let waste = this.focusTracker.secondsCapped * this.focusTracker.focusGen;
+    if (this.focusTracker.generatorCasts) {
+      Object.keys(this.focusTracker.activeFocusWasted).forEach((generator) => {
+        waste += this.focusTracker.activeFocusWasted[generator];
+
+      });
+    }
+    return Math.round(waste);
+  }
+
+  statistic() {
+    const totalFocusWaste = this.getTotalWaste();
+    const tooltipData = "You wasted <b>" + totalFocusWaste + " </b> focus. <br /> That's <b>" + Math.round(totalFocusWaste/(this.owner.fightDuration/1000 * this.focusTracker.focusGen) * 10000)/100 + "% </b> of your total focus generated. <br /> For more details, see the Focus Chart tab.";
```

Comment:
I think a template string would make this cleaner, also please use formatPercentage instead of doing this manually;

```jsx
const tooltipData = `You wasted <b>${totalFocusWaste}</b> focus.<br />
That's <b>${formatPercentage(totalFocusWaste / (this.owner.fightDuration / 1000 * this.focusTracker.focusGen) * 100)}% </b> of your total focus generated.<br />
For more details, see the Focus Chart tab.`;
```